velox/common/caching/AsyncDataCache.cpp

```diff
@@ -644,6 +644,9 @@ AsyncDataCache** AsyncDataCache::getInstancePtr() {
 }
 
 void AsyncDataCache::shutdown() {
+  if (ssdCache_) {
+    ssdCache_->shutdown();
+  }
   for (auto& shard : shards_) {
```

Comment:
Let's shutdown cache shards_ first and then shutdown ssdCache_ as logically, here is dependency from in-memory shard on ssd cache. Thanks!